### DISCUSSION TITLE
Contracts with All Quest Objectives

### DISCUSSION THREAD

**Discussion by Ghostdreamer11** [Aelto/W3_RandomEncounters_Tweaks#123] [General]
Hello,

I am not sure what way there might be around this (other than just disabling all quest objectives -or- contracts) . I had to install the bootstrap with the unmodified world map, but I think that may be stopping the contracts from appearing on the map after accepting them. Am I missing something, or is this just something that I will have to deal with?

Thank You for your time!

**Comment by Aelto** [comment]
Hello!

What's your issue exactly? The markers for the contracts don't appear on your map? Can you confirm the settings are all set to enable the map markers, if i remember correctly they're in the `optional features` menu. I've always used the unmodified world map as well and never had any issues while developing the mod, so i can confirm it is not the cause of the problem here!

**Comment by Ghostdreamer11** [comment]
Correct, the contract locations aren't appear on my map.

Awesome, at least it isn't that! I can confirm that in the optional features, all of the markers are "on"! I was able to stumble-upon the contract area, and it did outline it on the minimap. But when I checked the main map, it still did not display :(

**Reply by Aelto** [reply]
That's interesting. Based on what you said, my next guess would be that you did not merge the `sharedutils_mappins` entirely. You may have merged the `r4player.ws` changes it brings since you game launched (it wouldn't otherwise), but there is a chance you did not do it for `mapMenu.ws`.

Check the file in the script merger and confirm it is merged. If it doesn't appear at all then it means sharedutils are the only one modifying it and there is no need for merging.

**Reply by Ghostdreamer11** [reply]
Alright, so I ran the script merger originally, and it had me merge component.ws and mapMenu.ws . I did so, and then closed it. When I re-opened it to check what you suggested, it popped up a warning about the merged files not be loaded in the correct order. I went ahead and clicked "OK" for it to set the loading, and then started up the game. This seems to have fixed it! Thank you very much for your help, pointing me in the right direction.

PS. not sure how to mark this as complete/closed/or something similar.
